deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -406,7 +406,14 @@ handle_tick(QName,
                                {messages_persistent, M}
 
                                | infos(QName, ?STATISTICS_KEYS -- [consumers])],
-                      rabbit_core_metrics:queue_stats(QName, Infos),
+                      {ok, Queue} = rabbit_amqqueue:lookup(QName),
+                      Up = case ra:consistent_query(amqqueue:get_pid(Queue), fun (_) -> ok end) of
+                          {ok, ok, _} -> 1;
+                          _ ->
+                              rabbit_log:debug("ra:consistent_query failed for queue ~p~n", [QName]),
+                              0
+                      end,
+                      rabbit_core_metrics:queue_stats(QName, [{up, Up} | Infos]),
```

Comment:
Were other names considered? All this tells us is that a quorum queue does or does not have an online quorum.

Reply:
Yes, see my comment in the PR under Open questions. Instead of `rabbitmq_quorum_queues_up`, we can call the metric `rabbitmq_quorum_queues_consistent`?

Reply:
However, if the quorum queue doesn't have an online quorum, I think it's good to report this queue to be "down" since from the client perspective we can't write to it.